.github/workflows/scenarios-permian.yml

```diff
@@ -133,6 +137,9 @@ jobs:
           elif [ "${{ matrix.scenario }}" == "rhel9" ]; then
             echo "installation_tree=http://download.eng.bos.redhat.com/rhel-9/nightly/RHEL-9/latest-RHEL-9.4.0/compose/BaseOS/x86_64/os" >> $GITHUB_OUTPUT
             echo "modular_url=http://download.eng.bos.redhat.com/rhel-9/nightly/RHEL-9/latest-RHEL-9.4.0/compose/AppStream/x86_64/os" >> $GITHUB_OUTPUT
+          elif [ "${{ matrix.scenario }}" == "rhel10" ]; then
+            echo "installation_tree=http://download.eng.bos.redhat.com/rhel-10/nightly/RHEL-10/latest-RHEL-10.0.0/compose/BaseOS/x86_64/os" >> $GITHUB_OUTPUT
+            echo "modular_url=http://download.eng.bos.redhat.com/rhel-10/nightly/RHEL-10/latest-RHEL-10.0.0/compose/AppStream/x86_64/os" >> $GITHUB_OUTPUT
```

Comment:
You should use these for RHEL-10 Public Beta testing:
`http://download.eng.bos.redhat.com/rhel-10/nightly/RHEL-10-Public-Beta/latest-RHEL-10.0/compose/BaseOS/x86_64/os/` and 
`http://download.eng.bos.redhat.com/rhel-10/nightly/RHEL-10-Public-Beta/latest-RHEL-10.0/compose/AppStream/x86_64/os/`

Reply:
Good catch, thank you.

Reply:
Updated